src/init/element/templates/polymer-2.x/index.html

```diff
@@ -2,15 +2,15 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
-
+    <meta http-equiv="refresh" content="0;url=demo/" />
     <title><%= name %></title>
-
-    <script src="../webcomponentsjs/webcomponents-loader.js"></script>
-
-    <link rel="import" href="../iron-component-page/iron-component-page.html">
   </head>
   <body>
-    <iron-component-page src="<%= name %>.html"></iron-component-page>
+    <!--
```

Comment:
See `<meta>` tag above, when served / viewed in a browser will immediately redirect to the demo. This comment is for anyone viewing in their editor.